undertow-testing/build.gradle

```diff
@@ -0,0 +1,17 @@
+// Avoid including test dependencies from this module in
+// gradle module metadata.
+versionsLock {
+    testProject()
+}
+
+dependencies {
+    api 'org.junit.jupiter:junit-jupiter'
+    api 'io.undertow:undertow-core'
+    api 'io.undertow:undertow-servlet'
```

Comment:
```suggestion
    api 'io.undertow:undertow-servlet', {
        // Use  standard servlet-api instead
        exclude group: 'org.jboss.spec.javax.servlet', module: 'jboss-servlet-api_4.0_spec'
        exclude group: 'org.jboss.spec.javax.annotation', module: 'jboss-annotations-api_1.2_spec'
    }
    api <servlet-api>
    implementation <annotation-api>
```

Reply:
Done. It was actually api_1.3_spec....I don't know how anyone is expected to remember these package names.